Dockerfile.aspnet8-run

```
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-jammy-amd64 AS base
+RUN apt-get update && apt-get install -y curl
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)